ImageClassificationTests/CreateNoteVCTests.swift

```diff
@@ -18,22 +18,22 @@ class CreateNoteVCTests: XCTestCase {
         
     }
     
-    ///Test if the handle new note is successful by looking for a variable that represents success
+    ///Test if the handle new note is successful by looking for a variable that represents success as a result
     func testHandleNewNoteSuccess() {
         
     }
     
-    ///Test if the handle new note is a failure by looking for a variable that represents failure
+    ///Test if the handle new note is a failure by looking for a variable that represents failure as a result
     func testHandleNewNoteFailure() {
         
     }
     
-    ///Test if the handle update note is successful by looking for a variable that represents success
+    ///Test if the handle update note is successful by looking for a variable that represents success as a result
     func testHandleUpdateNoteSuccess() {
         
     }
     
-    ///Test if the handle update note is a failure by looking for a variable that represents failure
+    ///Test if the handle update note is a failure by looking for a variable that represents failure as a result
     func testHandleUpdateNoteFailure() {
         
     }
```

Comment:
This is a great example of testing both what happens on success and on failure 👏